test/test_restful.py

```diff
@@ -367,7 +368,7 @@ def setUpClass(cls):
         cls.auth_tag = crypto.do_hmac(cls.K, tenant_templ.agent_uuid)
 
         # Prepare policies for agent
-        cls.tpm_policy = '{"22":["0000000000000000000000000000000000000001","0000000000000000000000000000000000000000000000000000000000000001","000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000001","ffffffffffffffffffffffffffffffffffffffff","ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff","ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff"],"15":["0000000000000000000000000000000000000000","0000000000000000000000000000000000000000000000000000000000000000","000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000"]}'
+        cls.tpm_policy = "{}"
```

Comment:
Why the policy change here?

Reply:
To simplify the test using the empty policy. I can revert this if you prefer.

Reply:
No preference, just curiosity :)

Reply:
Good security says that a policy should be empty when there is none.  While a policy of all zeros or all ones likely cannot be satisfied, empty is easier to test and validate.

I.e., it's the difference between
if (empty)
and
if (empty or all zeros or all ones)